### ISSUE TITLE
Support cache busting

### ISSUE THREAD

**Issue by rnk** [rnk/django-media-bundler#1]
A timestamp should be added to the generated filename or a query parameter to the URL to force the browser to refetch the bundle if it updates.  I kind of like only having one file around and just diffing against it when the bundle is generated, so I'm leaning towards the query parameter.  It's still annoying because we have to track the version, and I'm not sure how to do that easily.

**Comment by peritus** [comment]
You could use a hash instead of a timestamp..

**Comment by rnk** [comment]
Yeah, I looked over at django-compress and that's what they do.  I just pushed a first pass at adding it in.  Unfortunately, the whole project needs tests, so all I can say is that it "works for me" so far.

**Comment by jneves** [comment]
I changed the behaviour to avoid the file copies and maintain the cache busting behaviour:

https://github.com/Leftfield/django-media-bundler/commit/66af89ecebae20d1dbeeeefb1c2538bc4cc8dc28

Feel free to pull.
